tests/utils.py

```diff
@@ -151,6 +153,33 @@ def validate_blocks_per_nodes(block_map, from_layer, to_layer, layers_per_epoch,
     print("\nvalidation succeeded!\n")
 
 
+def validate_beacons(log_messages):
+    epoch_messages = defaultdict(dict)
+
+    assert len(log_messages) > 0, f"no log messages"
+
+    for log in log_messages:
+        if log.epoch_id not in epoch_messages:
+            epoch_messages[log.epoch_id] = dict()
+
+        if log.beacon not in epoch_messages[log.epoch_id]:
+            epoch_messages[log.epoch_id][log.beacon] = 0
+
+        epoch_messages[log.epoch_id][log.beacon] += 1
+
+        assert log.beacon != '0x0000000000000000000000000000000000000000000000000000000000000000', \
+            f"beacon in epoch {log.epoch_id} is 0x00...00: {log.beacon}"
+
+        assert log.beacon != '0xe3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855', \
```

Comment:
how is it that this is the same beacon in every run? shouldn't it be a hash of all ATXs?

Reply:
The beacon should not be equal to `0xe3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855`, because this number is `sha256("")`